### ISSUE TITLE
First-time install: executing the api/users/show fails with 401

### ISSUE THREAD

**Issue by sdvinay** [servant-app/JAWS#21]
I ran through the installation guide.  I tried running the api/users/show locally first and it fails with 401 (I didn't install the JAWS client globally yet, but I don't expect that to be a problem):
```
$../../../cli/bin/jaws run
****** JAWS: Running Lambda Function Locally...
****** JAWS: Lambda Returned An Error:
{ status: 401, message: 'Invalid or expired access token' }
```

I went back and checked the steps from the installation guide; I updated the .env file with my own values (I made up JWT_ISSUER and JWT_SECRET, and copied in AWS_ADMIN_KEY, AWS_ADMIN_SECRET_KEY, AWS_LAMBDA_ROLE), and I updated the model_user.js with my table name.  The only suspicious step was that when creating the role ("jaws_lambda_role"), the wizard asked me to select a "role type"; the installation guide didn't mention this step, so I guessed that I should use "AWS Lambda".

Anything else I should check, or that I can try to help troubleshoot?

**Comment by ac360** [comment]
When you run a Lambda function in your JAWS API, the data in that function's event.json file is submitted to the Lambda function.  event.json should contain url params, url query strings and a JSON body submitted via a POST or PUT http request.  -- I'm going to add this in comments in each of the event.json file and write more information to help you and everyone understand this tonight.

The event.json in the users/show function includes an `access_token`.  This `access_token` is a JSON Web Token and it was left over from one of my own user records when I was developing JAWS.  In short, the access token you are using does not pertain to any user records in your database.

You need to first follow the DynamoDB section in the Installation guide, run the user/sign-up function to create a user in your own DynamoDB Users table, then take the jwt/access token that is received when you sign up a new user and put it in the event.json to use with the users/show function.  

Lots of helpful information on this process will be published in the next few hours.

Also, I will update the Installation guide to include that "Role Type" step.  Sorry about that!
